tests/test_server.py

```diff
@@ -52,140 +71,152 @@ def test_add_device(client, name='steelseries-rival310'):
     data = {
         'shortname': name
     }
```

Comment:
you should probably in a future PR replace this simple description by the json object we discussed previously that would ensure the creation of the device